src/main/kotlin/graphics/scenery/volumes/Volume.kt

```diff
@@ -591,7 +592,7 @@ open class Volume(
 
         @JvmStatic
         @JvmOverloads
-        fun <T : NumericType<T>> fromRAI(
+        fun <T : RealType<T>> fromRAI(
```

Comment:
I know this is not your fault, but while we're at it – please add Kdoc here :)

Reply:
But that's a class from the imgLib2 package, nothing scenery-related.